app/views/posts/show.html.erb

```diff
@@ -1 +1,34 @@
-<h1>Here are the details of the Post</h1>
+<html>
+  <head>
+    <%= stylesheet_link_tag 'posts_css' %>
+  </head>
+
+  <body>
+    <header>
+      <h1>Here are the details of the Post</h1>
+    </header>
+    <main class="flex-column">
+      <div class="post-detail-show">
+        <div class="post-detail-header-show flex-row">
+          <h2 class="post-title-show">
+            Post #<%= @post.id %> by <%= @post.author.name %>
+          </h2>
+          <p class="post-counters">
+            Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %>
+          </p>
+        </div>
+        <p><%= @post.text %></p>
+      </div>
+      <h2>List of Comments:</h2>
+      <% if @post.comments.any? %>
```

Comment:
- [ ] Likewise, please make sure that you app's design matches the given wireframe 💪 I think it would be better to remove the following highlighted according to the given wireframe. what do you think ? 😀

Current | Given 
---| ---|
![image](https://github.com/oovillagran/blog_app/assets/106119105/446a00de-c323-4d09-9b3e-e7d897887feb) | ![image](https://github.com/oovillagran/blog_app/assets/106119105/075d26f3-b940-445c-a080-e0f0521cb680)

Reply:
Done!